lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.cpp

```diff
@@ -344,6 +366,7 @@ namespace Js
         JavascriptString* methodName = JavascriptString::FromVar(methodNameProperty);
         int argumentsCount = TaggedInt::ToInt32(argumentsCountProperty);
 
+        BOOL staticMethod = JavascriptConversion::ToBoolean(staticMethodProperty, scriptContext);
         BOOL forceInline = JavascriptConversion::ToBoolean(forceInlineProperty, scriptContext);
```

Comment:
May want to add `Assert(Is())` for staticMethod and forceInline above (forceInline isn't your change but the remaining properties all have asserts). aliasProperty also doesn't have an assert, but I can't see where its used?

Reply:
I intended it as an optional property - if it's undefined ToBoolean will give false which is intended so an Assert here would be wrong. aliasProperty is used for the array iterator only (hence also optional) so again should not be asserted.